pkgs/by-name/cr/creality-print/package.nix

```
@@ -0,0 +1,70 @@
+{ lib
+, stdenv
+, undmg
+, fetchurl
+, makeDesktopItem
+, appimageTools
+,
+}:
```

```suggestion
{
  lib,
  stdenv,
  undmg,
  fetchurl,
  makeDesktopItem,
  appimageTools,
}:
```
Just a small nit, we should format this as per RFC 0166; we can use `nixfmt-rfc-style` from nixpkgs to automatically do this.

---

oh, I have that as my default formatter, I was running `nixpkgs-fmt` because I thought that's what it's used for nixpkgs, I'll fix that right up